app/views/documents/show/_scheduling_form.html.erb

```diff
@@ -5,12 +5,18 @@
 <% selected_time = submitted_values&.fetch("time") ||  current_scheduled_publish_datetime&.strftime("%l:%M%P")&.strip %>
 
 <%= form_tag(save_scheduled_publishing_datetime_path(@edition.document)) do %>
+
+  <% sheduleding_title = capture do %>
```

Comment:
Just a small typo here, should be `scheduling_title`

Reply:
I would say more than small. rushing is not good..
Thanks! Updated